File: C/Calculator.c
#include <stdio.h>

int main()
{
    printf("Menu\n\n");
    printf("1 - Addition\n");
    printf("2 - Subtraction\n");
    printf("3 - Multiplication\n");
    printf("4 - Division\n");
    printf("5 - Remainder Calculation\n");
    printf("6 - Larger out of two numbers\n");
    printf("0 - Exit\n");
    int a, b, c;
    do
    {
        printf("\nEnter your choice\n");
        scanf("%d", &c);
        if (c == 0)
        {
            break;
        }
        printf("Enter two numbers\n");
        scanf("%d%d", &a, &b);
        switch (c)
        {
        case 0:
            break;

        case 1:
            printf("%d + %d = %d\n", a, b, (a + b));
            break;

        case 2:
            printf("%d - %d = %d\n", a, b, (a - b));
            break;

        case 3:
            printf("%d x %d = %d\n", a, b, (a * b));
            break;

        case 4:
            printf("%d / %d = %d\n", a, b, (a / b));
            break;

        case 5:
            printf("%d modulo %d = %d\n", a, b, (a % b));
            break;

        case 6:
            printf("Larger out of %d and %d is %d.\n", a, b, (a > b) ? a : b);
            break;

        default:
            printf("Invalid Input. Enter correct option.\n");
        }

    } while (c != 0);

    return 0;
}

File: Python/Calculator.py
a=input("Enter 1st number : ")
b=input("Enter 2nd number : ")

print("\nMenu\n\n")
print("1 - Addition\n")
print("2 - Subtraction\n")
print("3 - Multiplication\n")
print("4 - Division\n")
print("5 - Modulo\n")
choice=input("\nEnter your choice : ")  # It takes input as string

if choice=='1' :
    sum=int(a)+int(b)
    print(a+" + "+b+" = "+str(sum))
elif choice=='2' :
    diff=int(a)-int(b)
    print(a+" - "+b+" = "+str(diff))
elif choice=='3' :
    pro=int(a)*int(b)
    print(a+" x "+b+" = "+str(pro))
elif choice=='4' :
    quo=int(a)/int(b)
    print(a+" / "+b+" = "+str(quo))
elif choice=='5' :
    mod=int(a)%int(b)
    print(a+" % "+b+" = "+str(mod))
else :
    print("Invalid Input")